Cilsil/Sil/Types/Struct.cs

```diff
@@ -16,31 +16,31 @@ public class Struct
         /// The instance fields of the type.
         /// </value>
         [JsonProperty]
-        public IEnumerable<FieldIdentifier> InstanceFields { get; }
+        public List<FieldIdentifier> InstanceFields { get; }
```

Comment:
We need List; IEnumerable and other Interfaces actually cause the "Auto" JSONSerializer  setting to add the "$type" field throughout the JSON. The JSONSerializer seems to struggle with what gets automatically inlined in $type for IEnumerable, so the deserialization then fails; additionally, the tenv gets inflated because IEnumerable would show up everywhere, with the $type for them.